scripts/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  #  Create a service named db.
+  db:
+    image: "postgres:12-alpine"
+    container_name: "pipeline_db"
+    #   Setup the username, password, and database name
+    environment:
+      - POSTGRES_USER=pipeline
+      - POSTGRES_PASSWORD=pipeline
+      - POSTGRES_DB=pipeline
```

Comment:
I suggest we omit the actual values from the `compose`-file and use [`host-specific`](https://docs.docker.com/compose/compose-file/#environment) definitions instead.

It would be also nice to provide default `.env`-file with default values for all the required environmental variables.